common/stdio.c

```diff
@@ -8,6 +8,8 @@
  * Paolo Scaffardi, AIRVENT SAM s.p.a - RIMINI(ITALY), arsenio@tin.it
  */
 
+#define DEBUG
```

Comment:
do we want to leave debug on?

Reply:
@Tim-Anderson oops, sorry, this shouldn't have been included in the PR. Fixed :)